build/pipelines/templates/build-console-ci.yml

```diff
@@ -2,9 +2,11 @@ parameters:
   configuration: 'Release'
   platform: ''
   additionalBuildArguments: ''
+  minimumExpectedTestsExecutedCount: 10  # Sanity check for minimum expected tests to be reported
+  rerunPassesRequiredToAvoidFailure: 5
 
 jobs:
-- job: Build${{ parameters.platform }}${{ parameters.configuration }}
+- job: BuildJob

```

Comment:
This also works? Wow, I know jack-all about azdo yaml. Didn't we need to use this for the dependency statements in the packaging tasks? Or is that outmoded because we're using _phases_ now? or whatever theyre called...

Reply:
Uhhh I don't think I intended to change that.

And I don't think I validated that the packaging tasks work with my runs. Don't they only operate in release mode?

Reply:
Changed it back.